bot/beltbot.py

```diff
@@ -66,38 +67,36 @@
 
 
 @BOT.command(name="request", help=_request_help)
-async def request_handler(ctx, colour, *body):
+async def request_handler(ctx, *, body):
```

Comment:
```suggestion
async def request_handler(ctx, *, request):
```

Reply:
I think rename to request as it better matches what is being referenced now that the entire request is in that kwarg